var/spack/repos/builtin/packages/ufs-weather-model/package.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class UfsWeatherModel(CMakePackage):
+    """The Unified Forecast System (UFS) Weather Model (WM) is a prognostic
+    model that can be used for short- and medium-range research and
+    operational forecasts, as exemplified by its use in the operational Global
+    Forecast System (GFS) of the National Oceanic and Atmospheric
+    Administration (NOAA)."""
+
+    homepage = "https://ufs-weather-model.readthedocs.io/en/latest/"
+    url      = "https://github.com/ufs-community/ufs-weather-model/archive/refs/tags/ufs-v1.1.0.tar.gz"
+    git      = "https://github.com/ufs-community/ufs-weather-model.git"
+
+    maintainers = ['t-brown']
+
+    version('2.0.0', tag='ufs-v2.0.0', submodules=True)
+    version('1.1.0', tag='ufs-v1.1.0', submodules=True)
+
+    variant('32bit', default=True, description='Enable 32-bit single precision arithmetic in dycore')
+    variant('avx2', default=False, description='Enable AVX2 instructions')
+    variant('build_type', default='Release', description='CMake build type',
+            values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel'))
```

Comment:
I noticed you override this variant in all packages. Is there a reason for this? If you want to always default to `Release` for all CMake packages you can set this in your `packages.yaml`. There's been talk of changing the default in the past.

Reply:
No reason, apart that I prefer `Release`, happy to remove the block and be standard then set it in my `packages.yaml`.

Reply:
It would probably be better to go with the default for now. You can chime in on #22918 and voice your opinion.